content/whats-next/whats-next-mid-term.json

```diff
@@ -1,17 +1,22 @@
 [
   {
-    "title": "SKU generator",
-    "description": "We saw a problem here. And we are thinking about it. If interested, don't hesitate to reach out to your customer success manager.",
-    "areas": ["Productivity"]
+    "title": "Product identifier generator",
+    "description": "We want the PIM to be responsible for creating unique product identifiers on the fly when importing/creating new products, based on nomenclatures.",
+    "areas": ["Automation","Governance"]
   },
   {
-    "title": "SKU generator",
-    "description": "We saw a problem here. And we are thinking about it. If interested, don't hesitate to reach out to your customer success manager.",
-    "areas": ["Productivity"]
+    "title": "Several product identifiers for one product",
+    "description": "We want to allow the use of several identifiers because in most of the cases, both EAN and SKU are needed. Some of you even use, in addition to those two, an internal identifier.",
```

Comment:
```suggestion
    "description": "We want to allow the use of several identifiers because in most of the cases, both GTIN and SKU are needed. Some of you even use, in addition to those two, an internal identifier.",
```

Reply:
We may talk about "GTIN" codes instead of EAN, as "EAN" stands for Europe only (it's UPC in the US for instance)